### ISSUE TITLE
Issues in link of Cross Entropy and About Me in the Contents Section

### ISSUE THREAD

**Issue by intuitifai** [rdipietro/jupyter-notebooks#9]
I am not sure if everyone can reproduce this issue, but in the `Content section` for `Cross Entropy and About Me`, I am not able to open the links
Maybe you can give links like:
### Contents
`...`
```
* [Cross Entropy](https://rdipietro.github.io/friendly-intro-to-cross-entropy-loss/#Cross-Entropy)
* [About Me](https://rdipietro.github.io/friendly-intro-to-cross-entropy-loss/#About-Me)
```
`...`

and it should work like a charm, something like:
### Contents
`...`
* [Cross Entropy](https://rdipietro.github.io/friendly-intro-to-cross-entropy-loss/#Cross-Entropy)
* [About Me](https://rdipietro.github.io/friendly-intro-to-cross-entropy-loss/#About-Me)  

`...`

Actually I believe that you can try giving hyperlinks in markdown using the same structure, i.e, instead of using `<url>/cross-entropy`, use `<url>/Cross-Entropy` (usage of same letters as in text, and separating them with hyphens) and it should work for all links in Contents Section
